cmd/root.go

```diff
@@ -138,6 +140,7 @@ func init() {
 	rootCmd.Flags().StringVarP(&separator, "separator", "s", ",", "use specific separator (\\t, or any one character string)")
 
 	rootCmd.Flags().BoolVarP(&timed, "time", "", false, "Measure time")
+	rootCmd.Flags().BoolVarP(&lazyQuotes, "lazyquotes", "", false, "allow unescaped quotes")
```

Comment:
I guess we don't need to do a `BoolVarP` if there is no shorthand. We can just use `BoolVar`.

Reply:
Ok, I made this change in 2e5a8d1.